.github/workflows/pull-request.yml

```diff
@@ -341,6 +341,7 @@ jobs:
             cat .env .env.temp .env.staging > .env.prod
 
             SSH_PASSWORD=${{ secrets.SSH_PASSWORD }} sh scripts/pullrequest/create.sh
+            PRODUCTION_PG_URL_FOR_USER_GITHUB_ACTIONS=${{secrets.PRODUCTION_PG_URL_FOR_USER_GITHUB_ACTIONS}} sh scripts/sync-users.sh
```

Comment:
thanks for this! 

one initial comment: if we only sync users from production onto pizzas, I believe editor authentication will still fail because pizza's use _staging_ google oauth. so I think we first need to sync production to staging (so I'd expect to see this command also called from `.github/workflows/sync-staging-db.yml`), and then production/staging to pizzas. 

does that make sense? or is there another reason i'm missing that you're able to skip the sync to staging here? 

once updated with a successful vult build, happy to test and approve!

Reply:
The users table doesn't contain any oauth data so I think we're good to sync down from production into both staging and pizzas.

Reply:
Well then again, there's the issue that user ids need to match. You're right, best is to do prod->sta, then sta->pizza